src/public/js/video-tagging/video-tagging.html

```diff
@@ -788,7 +788,7 @@
             this.editor.RM.redrawAllRegions();
         }
         //Clears areaSelector state when moving across frames
-        this.editor.RM.onManipulationEnd();
+        this.editor.RM.redrawAllRegions();
```

Comment:
@PIC123 did you mean to include this on this PR?

Reply:
Hey, yah that was causing issues with loading so I just wanted to update the small thing for release. I can remove it as well

Reply:
That's fine with me :)